src/pure/stack_set.rs

```diff
@@ -563,6 +563,38 @@ where
         self.update_previous_tag(self.screens.focus.workspace.tag.clone());
         self.screens.focus_up();
     }
+    
+     ///Move focus to next tag
+     pub fn next_tag(&mut self) {
+        let tags = self.ordered_tags();
+        if tags.len() == 0 {
+            return;
+        }
+        let i = tags
+            .iter()
+            .position(|a| a == self.current_tag())
+            .unwrap_or_default();
+
+        let new_i = (i + 1) % tags.len();
+        let new_tag = &tags[new_i];
+        self.focus_tag(new_tag);
+    }
+
+    ///Move focus to previous tag
```

Comment:
Can you add a leading space to doc comments please and also add a quick explanation about the semantics of next/previous tag given that it might not be immediately obvious to users as this ordering is derived from workspace ID (so effectively creation order) rather than numeric or alphabetical order.